### PR TITLE
add fpocket

### PR DESCRIPTION
**Pull request by simonbray**

### CODE REVIEW

File: chemicaltoolbox/fpocket/fpocket.xml [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+    <description>- find potential binding sites in protein structures</description>
+    <requirements>
+        <requirement type="package" version="3.1.3">fpocket</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+
+        ln -s '$input' ./input.pdb &&
+        fpocket
+            -f ./input.pdb
+            #if $inp.pocket_type == 'channel':
```
Comment (bgruening) [inline chemicaltoolbox/fpocket/fpocket.xml]: just checking, `small_mol` is not needed here?

Comment (bgruening) [reply chemicaltoolbox/fpocket/fpocket.xml]: this could be a elif ...

Comment (simonbray) [reply chemicaltoolbox/fpocket/fpocket.xml]: no, small_mol is the default

Comment (simonbray) [reply chemicaltoolbox/fpocket/fpocket.xml]: will update to elif

**Pull request merged**